javascript/javascript1/week3/HW_2.js

```diff
@@ -0,0 +1,101 @@
+console.log('When will we be there??')
+const travelInformation = {
+  speed: 100,
+  destinationDistance: 432,
+}
+
+function counterTravelTime(object) {
+  const reminder = object.destinationDistance % object.speed
+  const minutes = Math.round((reminder * 60) / object.speed)
+  const hours = (object.destinationDistance - reminder) / object.speed
+  return `${hours} hours and ${minutes} minutes `
+}
+
+const travelTime = counterTravelTime(travelInformation)
+console.log(travelTime) // 8 hours and 38 minutes
+
+const travelInformation1 = {
+  speed: 50,
+  destinationDistance: 432,
+}
+
```

Comment:
this works, but there are several corner cases: if speed for whatever reason is 0 and most importantly if you get to your destination either in less than 1 hr or in precisely x hours. '0 hrs' or '0 minutes' are not really a thing. 
This doesn't break your code, but the lesson is: corner cases matter!